sdk/nodejs/tests/runtime/langhost/run.spec.ts

```diff
@@ -1220,6 +1220,14 @@ describe("rpc", () => {
                 return { urn: makeUrn(t, name), id: undefined, props: undefined };
             },
         },
+        "ambiguous_entrypoints": {
+            program: path.join(base, "069.ambiguous_entrypoints"),
+            expectResourceCount: 1,
+            expectedLogs: {
+                count: 1,
```

Comment:
Can't this check the log message itself?

Reply:
I didn't see any mechanism for doing so, though I could see a case for adding that capability in this PR if you feel strongly enough about it.

Reply:
Nevermind!  I found places where we do exactly this.  I'll check for the actual message.